apps/api/e2e/utils/api-types.ts

```diff
@@ -658,3 +658,32 @@ export interface CompetitionLeaveResponse extends ApiResponse {
   success: true;
   message: string;
 }
+
+// Agent rank (global rankings)
+export interface AgentRank {
+  id: string;
+  name: string;
+  imageUrl?: string;
+  metadata?: AgentMetadata;
+  rank: number;
+  score: number;
+  numCompetitions: number;
+}
+
+// Global leaderboard response
+export interface GlobalLeaderboardResponse extends ApiResponse {
+  success: true;
+  stats: {
```

Comment:
and then we use the global `AgentRank` in our leaderboards response, along with global stats about all of the competitions (stats are needed for ui-related information)